src/base/exceptions.cpp

```diff
@@ -30,11 +30,10 @@
 
 RuntimeError::RuntimeError(const QString &message)
     : std::runtime_error {message.toUtf8().data()}
```

Comment:
I was looking at this earlier, IMO it can be `message.toStdString()`, do you mind change it too?

Reply:
Why do you think it is better?
std::exception still returns it as `const char *`. Won't it just be another conversion?
In addition, this constructor (from `const char *`) is newer (added in c++11). Apparently, there are some reasons for adding it.

Reply:
In addition, this is just one of the implementations, others may be different.
I don't see any particular advantage in either of these two options, so let it stay as it is.